test/utilities/miscellaneous.ts

```diff
@@ -81,14 +91,14 @@ export async function createCommand(
   folder: string,
   extension: string
 ): Promise<string | undefined> {
-  const workbench = await (await browser.getWorkbench()).wait();
-  await executeQuickPick('View: Clear Output', 1);
-  const inputBox = await executeQuickPick(`SFDX: Create ${type}`, 1);
+  const workbench = await getWorkbench();
+  await executeQuickPick('View: Clear Output', Duration.seconds(1));
```

Comment:
Import and use the helper function from `outputView.ts`

```suggestion
  await clearOutputView();
```